packages/lesswrong/components/sunshineDashboard/AdminMetadata.tsx

```diff
@@ -24,10 +24,11 @@ const adminMetadataQuery = gql`query AdminMetadataQuery {
 }`;
 
 const AdminMetadata = ({ classes }) => {
-  const { data: adminMetadata, loading } = useQuery(adminMetadataQuery, { ssr: true });
+  const { data, loading } = useQuery(adminMetadataQuery, { ssr: true });
```

Comment:
Can you explain to me how the change here is preventing a bug?

Reply:
It was destructuring the result wrong; the result had a layer of wrapper that was assumed to not be there. If we put it all on one destructuring line (ignoring the loading case), the correct version would have been

```
const { data: { AdminMetadata: adminMetadata }, loading } = useQuery(adminMetadataQuery, { ssr: true });
```

(The bug was introduced by me when converting this component from HoCs to hooks, and not noticed because this page is admin-only and rarely visited).

Reply:
This isn't very strongly related to soft filtering, I just happened to notice it when I was adding the tag-relevance index.

Reply:
Ah, I see. All makes sense. Thanks.